src/index.css

```diff
@@ -10,4 +10,29 @@ body {
 
 button:hover {
   cursor: pointer;
+}
+
+
+@media (width > 375px) {
+  .MuiDialog-paper {
+    border-radius: 10px !important;
+    height: 300px !important;
+    background-color: #000000 !important;
+  }
+}
+
+@media (768px <= width <= 1024px) {
+  .MuiDialog-paper {
+    border-radius: 10px !important;
+    height: 500px !important;
+    background-color: #000000 !important;
+  }
+}
+
+@media (width > 1024px) {
+  .MuiDialog-paper {
+    border-radius: 10px !important;
+    height: 500px !important;
+    background-color: #000000 !important;
+  }
 }
```

Comment:
better to not override styles like this, especially for ui component libraries. Check if Mui allows you to specify custom styles or classes.

Reply:
Removed the override of styles and used props which are provided by MUI to style as required